tests/qnodes/test_qnode_decorator.py

```diff
@@ -97,6 +98,25 @@ def circuit(a):
     assert isinstance(circuit, JacobianQNode)
     assert hasattr(circuit, "jacobian")
     assert circuit.h == step_size
+    assert circuit.order == 1
+
+
+@pytest.mark.parametrize("order", [1, 2])
+def test_finite_diff_qubit_qnode_passing_order_through_decorator(order):
+    """Test that a finite-difference differentiable qubit QNode is correctly created when
+    diff_method='finite-diff' and the order is set through the decorator."""
+    dev = qml.device('default.qubit', wires=1)
```

Comment:
It seems that this test is only testing the order; the creation of the finite difference QNode is likely done in a previous test.

Maybe slightly easier to read if this test is renamed `test_setting_order`? My comment applies to the following test as well, but that was an existing test name 😆

Reply:
Good idea, onboard with the change for both from my end as well if accepted! :) (the other one was coming from #530)

Reply:
dd168e4